src/generator/models.ts

```diff
@@ -716,3 +741,13 @@ function generateAliasType(structDef: StructDef, receiver: string, forMarshal: b
   text += `\t}\n`;
   return text;
 }
+
+// differentiates between objects and basic types defined in Go. Returns true if it finds an object to expand for the error message. 
+function isObjectType(schema: Schema): boolean {
```

Comment:
This already exists in `common/helpers.ts`, see `isObjectSchema()`

Reply:
Done. Removed the repeated function.